<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>image_editor</class>
 <widget class="QDialog" name="image_editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1145</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="adjust_image_group">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>Image</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="load_image">
            <property name="text">
             <string>Load Image</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="file_name">
              <property name="text">
               <string>(No File Selected)</string>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QComboBox" name="color_map">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="log_scale">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>log scale</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ImageTabWidget" name="image_tab_widget"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dialog_buttons">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageTabWidget</class>
   <extends>QTabWidget</extends>
   <header>image_tab_widget.py</header>
   <container>1</container>
   <slots>
    <slot>open_files()</slot>
    <slot>set_tabbed_view(bool)</slot>
    <slot>show_nav_toolbar(bool)</slot>
    <slot>show_toolbar(bool)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>dialog_buttons</sender>
   <signal>accepted()</signal>
   <receiver>image_editor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>594</x>
     <y>542</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
